forta-uniswap-py/README.md

```diff
@@ -0,0 +1,27 @@
+# Large Uniswap Detector Agent
+
+## Description
+
+This agent detects large uniswap transactions
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- AE-UNISWAP-LARGESWAP-METHODID
```

Comment:
Rename to "AE-UNISWAP-LARGESWAP-ETH"?  The next one could be called "AE-UNISWAP-LARGESWAP".